# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/CookieStore.h
# Commit: 30d5dfedf252
# Full Hash: 30d5dfedf252f4f10a0d8948e86f4e57f1c4ccf9
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-11 09:21:34
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 8 - CookieStore API - e10s support, r=smaug,cookie-reviewers,timhuang
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217574
# ==============================================================================

diff -r fc3b4c5ee4b2 -r 30d5dfedf252 dom/cookiestore/CookieStore.h
--- a/dom/cookiestore/CookieStore.h	Tue Sep 10 16:31:40 2024 +0000
+++ b/dom/cookiestore/CookieStore.h	Tue Sep 10 16:31:41 2024 +0000
@@ -17,6 +17,7 @@
 
 class CookieData;
 class CookieStoreChild;
+class CookieStoreNotificationWatcherWrapper;
 class CookieStoreNotifier;
 class Promise;
 
@@ -72,6 +73,7 @@
 
   RefPtr<CookieStoreChild> mActor;
   RefPtr<CookieStoreNotifier> mNotifier;
+  RefPtr<CookieStoreNotificationWatcherWrapper> mNotificationWatcher;
 };
 
 }  // namespace mozilla::dom